support/cas-server-support-oidc/src/test/java/org/apereo/cas/oidc/token/OidcRegisteredServiceJwtAccessTokenCipherExecutor2Tests.java

```diff
@@ -0,0 +1,51 @@
+package org.apereo.cas.oidc.token;
+
+import org.apereo.cas.oidc.AbstractOidcTests;
+
+import lombok.val;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.cloud.context.scope.refresh.RefreshScope;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * This is {@link OidcRegisteredServiceJwtAccessTokenCipherExecutor2Tests}.
+ *
+ * @author Misagh Moayyed
+ * @since 6.1.0
+ */
+@Tag("OIDC")
+public class OidcRegisteredServiceJwtAccessTokenCipherExecutor2Tests extends AbstractOidcTests {
```

Comment:
I have to split this from OidcRegisteredServiceJwtAccessTokenCipherExecutorTests, because forcing changing the key in this way would cause other test cases in OidcRegisteredServiceJwtAccessTokenCipherExecutorTests to fail.